### ISSUE TITLE
Crash when adding to config

### ISSUE THREAD

**Issue by SigurdCole** [thommcgrath/Beacon#53]
First off, this utility is amazing, thanks so much for all your hard work!

I'm playing single player, so I've tweaked the settings and configs pretty extensively. So far I haven't had any problems, but when I tried to add the lines from Beacon's Game.ini to my existing one, ARK crashes - doesn't even hit the title screen. Even adding a single line makes a crash. 

I've thrown my config into Pastebin [here](https://pastebin.com/smN2kGWR). Any pointers?

**Comment by thommcgrath** [comment]
I'm sorry, no, I cannot see any problems.

**Comment by SigurdCole** [comment]
Is there anything else that would help track it down? Or is it just a mystery?

**Comment by RudiVuuren** [comment]
This started happening to me as soon as the new update for Beacon was released. The application work fine until I regenerated the config.

Here is my Game.ini https://pastebin.com/TSavatDm
And here is my error: https://ibb.co/mrSgZa

**Comment by thommcgrath** [comment]
@SigurdCole, can you post a screenshot of your crash? I'm curious if it is the same.

**Comment by RudiVuuren** [comment]
@thommcgrath I did post it, but here is it again: https://ibb.co/mrSgZa

**Comment by thommcgrath** [comment]
Sorry @RudiVuuren, that was intended for the original reporter.

**Comment by RudiVuuren** [comment]
Sorry about that, thanks for your time and effort!

**Comment by SigurdCole** [comment]
Sure thing, [here's my error.](http://i.imgur.com/M6CZ5zn.png)

**Comment by thommcgrath** [comment]
Yep, same crash as everybody else. It doesn't even look to be loot related, as it's crashing while loading a material.

I have switched Beacon to use class strings again. No need to download a new build, just open your file and export again.

**Comment by SigurdCole** [comment]
Loaded smooth. I'll confirm actual beacon drops later, but no crashes! Thank you!

**Comment by RudiVuuren** [comment]
I got the exact same error when regenerating the ini file and pasting its contents in my game.ini file. I then all my configuration in my game.ini file and only pasted the generated code of Beacon. It finally launched. So after re-pasting my configuration AFTER Beacon's configuration, the game didn't crash. And yes, all my configuration works perfectly - for each beacon type, loot crates and bosses.

Thank you so much @thommcgrath!
